sdkproject/Assets/Mapbox/Unity/MeshGeneration/Modifiers/MeshModifiers/LineMeshModifier.cs

```diff
@@ -1,30 +1,83 @@
+using System;
+using Mapbox.Unity.Utilities;
+using System.Collections.Generic;
 using Mapbox.Unity.Map;
+using UnityEngine;
+using Mapbox.Unity.MeshGeneration.Data;
 
 namespace Mapbox.Unity.MeshGeneration.Modifiers
 {
-	using System.Collections.Generic;
-	using UnityEngine;
-	using Mapbox.Unity.MeshGeneration.Data;
-
 	/// <summary>
 	/// Line Mesh Modifier creates line polygons from a list of vertices. It offsets the original vertices to both sides using Width parameter and triangulates them manually.
 	/// It also creates tiled UV mapping using the line length.
-	/// MergeStartEnd parameter connects both edges of the line segment and creates a closed loop which is useful for some cases like pavements around a building block.
 	/// </summary>
 	[CreateAssetMenu(menuName = "Mapbox/Modifiers/Line Mesh Modifier")]
 	public class LineMeshModifier : MeshModifier
 	{
-		[SerializeField]
-		LineGeometryOptions _options;
+		#region Constants
+
+		private readonly float _cosHalfSharpCorner = Mathf.Cos(75f / 2f * (Mathf.PI / 180f));
+		private readonly float _sharpCornerOffset = 15f;
+		private readonly float _miterLimit = 0.2f;
+		private readonly float _roundLimit = 1.05f;
```

Comment:
These constants should be set from the `LineGeometryOptions _options` datacontainer.